### ISSUE TITLE
[Bug] OpenCV `cvUndistortPointsInternal`

### ISSUE THREAD

**Issue by Pk13055** [bytedeco/procamtracker#9]
Calibration is done using `procamcalib`. On running `procamtracker`, this error pops up. Is there some calibration param or setting that I am missing?

## System

- Distro: `Linux pk-pc 4.19.10-041910-generic #201812170433 SMP Mon Dec 17 09:35:34 UTC 2018 x86_64 x86_64 x86_64 GNU/Linux`
- OS: Ubuntu 18.10
- Java: 
```
openjdk version "1.8.0_191"                                              
OpenJDK Runtime Environment (build 1.8.0_191-8u191-b12-2ubuntu0.18.10.1-b
12)  64-Bit Server VM (build 25.191-b12, mixed mode)                  
```
- Stacktrace:

```java
SEVERE: OpenCV Error: org.bytedeco.javacpp.BytePointer[address=0x7f7a93d
7c60,position=0,limit=0,capacity=0,deallocator=null] (CV_IS_MAT(_distCoe
fs) && (_distCoeffs->rows == 1 || _distCoeffs->cols == 1) && (_distCoeff
->rows*_distCoeffs->cols == 4 || _distCoeffs->rows*_distCoeffs->cols == 
 || _distCoeffs->rows*_distCoeffs->cols == 8 || _distCoeffs->rows*_distC
effs->cols == 12 || _distCoeffs->rows*_distCoeffs->cols == 14))         
in function cvUndistortPointsInternal, /home/travis/build/javacpp-preset
/opencv/cppbuild/linux-x86_64/opencv-4.0.1/modules/calib3d/src/undistort
cpp(403)
```

**Comment by saudet** [comment]
Do you have the full stack trace of where that error occurs?

**Comment by saudet** [comment]
This should be fixed in 1.5.8, see https://github.com/bytedeco/javacv/issues/1802. Thanks for reporting!
